runelite-client/src/main/java/net/runelite/client/rs/ClientLoader.java

```diff
@@ -203,14 +205,24 @@ public Applet load()
 		catch (IOException | ClassNotFoundException | InstantiationException | IllegalAccessException
 			| CompressorException | InvalidHeaderException e)
 		{
+			String message = "Error loading the Old School RuneScape client";
 			if (e instanceof ClassNotFoundException)
 			{
 				log.error("Unable to load client - class not found. This means you"
 					+ " are not running RuneLite with Maven as the client patch"
 					+ " is not in your classpath.");
 			}
+			else if (e instanceof SocketTimeoutException)
+			{
+				log.error("Unable to load client - connection timed out. This is most"
```

Comment:
This is already being logged below so this makes no sense?

Reply:
I followed the existing format of a more in-depth log based on the error instance, you'll see that the `ClassNotFoundException` has a similar message right above this. Should I remove this or leave it in?